tools/frrcommon.sh.in

```diff
@@ -45,7 +45,7 @@ RELOAD_SCRIPT="$D_PATH/frr-reload.py"
 debug() {
 	[ -n "$watchfrr_debug" ] || return 0
 
-	printf '%s %s(%s):' "`date +%Y-%m-%dT%H:%M:%S.%N`" "$0" $$ >&2
+	printf '%s %s(%s):' "$(date +%Y-%m-%dT%H:%M:%S.%N)" "$0" $$ >&2
```

Comment:
Is this for readability? Not sure why you're replacing backticks… not opposed to it either, just wondering.

(This is a non-blocking comment)

Reply:
shellcheck claims ticks are "legacy"
https://github.com/koalaman/shellcheck/issues/364#issuecomment-97578330

I blindly do what tools tell me to